renovate.json

```diff
@@ -56,13 +56,13 @@
       "matchPackageNames": [
         "kindest/node"
       ],
-      "allowedVersions": "<=1.26"
+      "allowedVersions": "<=1.27"
```

Comment:
not sure if these are ever used (usually bump was done manually)

Reply:
true, we do it manually, I was also thinking about removing them but kept is just as a reminder.